pglast/printers/ddl.py

```diff
@@ -9,6 +9,8 @@
 from .. import enums
 from ..node import Missing, List, Node
 from ..printer import node_printer
+from ..enums.parsenodes import (OPCLASS_ITEM_OPERATOR, OPCLASS_ITEM_FUNCTION,
```

Comment:
That's fine for me, but given we already imported the whole `enums` subpackage above, maybe it's more convenient to just use `enums.FOO` down there, instead of importing each symbol.

Reply:
You're right, I didn't remember the whole parsenodes modules was available in enums. Done